jni/events/baseaudioevent.h

```diff
@@ -93,17 +93,17 @@ class BaseAudioEvent
         // this allows for high precision positioning, see positionEvent() for a
         // positioning example using musical concepts
 
-        virtual void setSampleLength( int value );
-        virtual void setSampleStart( int value );
-        virtual void setSampleEnd( int value );
+        virtual void setEventLength( int value );
+        virtual void setEventStart( int value );
+        virtual void setEventEnd( int value );
 
```

Comment:
Renamed these functions as their name was confusing when dealing with _SampleEvents_. These methods describe the events position and duration, not of its contents (e.g. of a _sample_ when dealing with a _SampleEvent_).